pylib/gyp/__init__.py

```diff
@@ -696,7 +696,7 @@ def main(args):
         return 1
 
 
-# NOTE: setuptools generated console_scripts calls function with no arguments
+# NOTE: console_scripts calls this function with no arguments
```

Comment:
I think the original comment is still correct and concise as this function is referenced in `pyproject.toml#project.scripts`?

Reply:
The Python Packaging Authortiy has respecified `console_scripts` to be supported beyond `setuptools` as discussed at
https://packaging.python.org/en/latest/specifications/entry-points